kgraphql/src/main/kotlin/com/apurebase/kgraphql/Extensions.kt

```diff
@@ -21,17 +21,15 @@ internal fun String.dropQuotes(): String = if (isLiteral()) {
 
 internal fun String.isLiteral(): Boolean = startsWith('\"') && endsWith('\"')
 
-internal fun KClass<*>.isIterable() = isSubclassOf(Iterable::class)
+fun KClass<*>.isIterable() = isSubclassOf(Iterable::class)
```

Comment:
We could think about having an extra sub-project for such extensions, to avoid making them part of the public api, even if we want to use them in multiple of our sub-projects.

Reply:
Hmm 🤔 I'd probably prefer to explicitly mark public API as such but I assume this will be quite some initial effort.